src/main/java/org/apache/mesos/scheduler/plan/BlockFactory.java

```diff
@@ -11,6 +12,6 @@
  * detected for a Task.
```

Comment:
I mean, which TaskSpecifications are supposed to be passed in a single call to `BlockFactory.getBlocks()`, in practice?

All TaskSpecifications across the service? Or just the TaskSpecifications from a single TaskSet? Or something else?

(I think this is something that a developer writing their own BlockFactory implementation would be wondering)